src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/template-types/.content.xml

```diff
@@ -8,4 +8,7 @@
     #if ( $includeForms == "y" or $includeFormsenrollment == "y" or $includeFormscommunications == "y" or $includeFormsheadless == "y" )
     <af-page-v2/>
     #end
+    #if ( $aemVersion == "cloud" and ( $includeForms == "y" or $includeFormsenrollment == "y" or $includeFormscommunications == "y" or $includeFormsheadless == "y") )
+    <afv2-fragment-page/>
+    #end
```

Comment:
Is fragment 2.0 applicable to the cloud only?

Reply:
Currently, till it gets sideported to release/650. I have raised PR for that as well in core components, once its merged, this check will go away.